Added for Android Build, use Java version 11 (#149)
mavlink/qgc-dev-guide: en/getting_started/README.md

- **Ubuntu:** 64 bit, gcc compiler
- **Windows:** Vista or higher, [Visual Studio 2019 compiler](#vs) (64 bit)
- **iOS:** 10.0 and higher

> - **Android:** Jelly Bean (4.1) and higher. Standard QGC is built against ndk version 19. Use Java JDK version 11 in Qt Creator to build

Reviewer: ```suggestion
- **Android:** Jelly Bean (4.1) and higher. Standard QGC is built against ndk version 19. Note Java JDK 11 is also required.
```
